src/status_im/ui/screens/browser/views.cljs

```diff
@@ -64,9 +65,15 @@
     [components/activity-indicator {:animating true}]]))
 
 (defn on-navigation-change [event browser]
-  (let [{:strs [url canGoBack canGoForward]} (js->clj event)]
-    (when (and (not (:dapp? browser)) (not= "about:blank" url))
-      (re-frame/dispatch [:update-browser (assoc browser :url url)]))
+  (let [{:strs [url canGoBack canGoForward loading]} (js->clj event)]
+    (when (not= "about:blank" url)
+      (let [browser2 (assoc browser :url url)]
+        (if (not loading)
+          (if (browser-history/forget-page-load?)
+            (do (re-frame/dispatch [:update-browser browser2])
```

Comment:
but you updated it already when set forget-page-load to true?

Reply:
Good catch, will fix :+1:

Reply:
Done!